components/search-tab-group/search-projects-tab.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Helmet } from "react-helmet";
+import ProjectLoader from '../../components/project-loader/project-loader.jsx';
+
+export default (props) => {
```

Comment:
nit: this seems almost identical to the profile tab component. any way we can generalize it? Something like this?

search-tab.jsx
```js
export default const createSearchTab = (Loader, type) => (props) => {
  const { keywordSearched } = props;
  return (
    <div className="{`search-${type.toLowercase()}-tab`}">
      <Helmet><title>{type} search</title></Helmet>
      { keywordSearched && <Loader search={keywordSearched} showCounter={true} />}
    </div>
  );
};
```